docs/admin/access.rst

```diff
@@ -165,9 +155,9 @@ The authentication models consist of several objects:
 Permission checking
 +++++++++++++++++++
 
-Whenever a permission is checked to decide whether one is able to perform a
-given action, the check is carried out according to scope, and the following
-checks are performed in this order:
+Whenever a permission is checked to decide whether the user is able to
+perform a given action, the check is carried out according to scope, and
```

Comment:
does the word "scope" have to be used?

Reply:
I think that is better, but I don't really know when the check is carried out.

Reply:
??

the check is performed when a user tries to do a certain action... or what do you mean?